ArticleTemplates/assets/scss/garnett-type/_immersive.scss

```diff
@@ -101,70 +100,83 @@
         }
 
         &__header-info {
-            background-color: rgba(0, 0, 0, .6);
+            background-color: color(shade-1);
+            color: color(shade-7);
             padding: base-px(.25, 1, 2, 1);
-            width: 100%;
+            margin: base-px(-6.5, 0, 0, 0);
+            position: relative;
+            z-index: 10;
+            width: calc(100% - 32px);
+            @include mq($to: mobileMedium) {
+                width: calc(100% - 24px);
+            }
 
-            @include mq($from: col3) {
-                padding-top: base-px(1);
+            @include mq($from: col2) {
+                width: auto;
+                padding-top: base-px(.5);
+            }
+
+            @include mq(col2, col3) {
+                margin-right: cols($base-3, 2);
+            }
+
+            @include mq(col3, col4) {
+                margin-right: cols($base-4, 6);
             }
 
             @include mq($from: col4) {
+                margin-left: auto;
+                margin-right: auto;
                 padding: {
-                    left: 0;
-                    right: 0;
+                    left: base-px(1);
+                    right: 340px;
+                }
+                width: 1200px;
+                transform: translateX(228px); //240 - 8
```

Comment:
🕵️  not quite sure what the numbers represent, but 240 - 8 is `232px`. Could it be worth assigning `228` to a variable as it's referenced in a few places which I imagine would all need updating if it were to change?

Reply:
🤣I dreamt of an 8 pt gutter on this template but immediately regretted it haha
Will update everything, thanks george!

Reply:
I've updated it to use the SASS calculation `240px - base-px(1)` instead, which I think is easier to understand — does that seem good to you?

Reply:
Sounds good to me